oneutil/etl/databento.py

```diff
@@ -1,15 +1,21 @@
 import os
+import re
+from datetime import date, datetime
+
+import boto3
 import databento
+import pandas as pd
+
 from oneutil.logging import logger
-from oneutil.etl.aws import read_s3_bucket_file
+from oneutil.etl.aws import read_s3_bucket_file, get_s3_bucket_files
 
 
 def get_df_from_s3(
-    filename: str,
-    bucket: str = "onesquared-databento",
-    region: str = "us-east-1",
-    public_key: str = os.environ.get("s3_public_key"),
-    private_key: str = os.environ.get("s3_private_key"),
+        filename: str,
+        bucket: str = "onesquared-databento",
+        region: str = "us-east-1",
+        public_key: str = os.environ.get("s3_public_key"),
+        private_key: str = os.environ.get("s3_private_key"),
```

Comment:
Is there a reason for this formatting change? When working in a team, try to avoid unnecessary format changes - it's a waste of resources.

I ran the code above through the `black` linter and it did not prompt this change. Did you use the same formatter? If not, please install it. VSCode has a plugin.